tests/keys/private.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Are we signing with a publicly available private key? We are not pushing the uki artifact when we release right?

Reply:
We currently are not pushing anything anywhere.

This is for testing mainly and done with public private keys and certs, all of them used in the uki test pipeline.

Final releases would need a specific workflow to sign and measure them with our private keys.

But that is like a whole other conversation on how to sign them, where, who has the key, how we cycle them, etc..

Not willing to open that discussion right now as it's a big one 🤣

Reply:
Sure, I'm just making sure I got this right.